build/BuildEnv.shade

```diff
@@ -33,4 +33,12 @@ functions
             return Environment.GetEnvironmentVariable("TEAMCITY_PROJECT_NAME") != null;
         }
     }
-  }
+    
+    bool IsBuildV2
+    {
+        get 
+        { 
+            return Environment.GetEnvironmentVariable("KOREBUILD_BUILD_V2") == "1"; 
```

Comment:
"V2" isn't helpful and other changes to `dnu` use are controlled by `"NUGET3_build_options"`, `""NUGET3_pack_options"`, and `"NUGET3_restore_options"` environment variables. Since those names are no longer where we're heading, suggest `"DNU_BUILD_ONE_SHOT"` for this and to rename the existing environment variables to start with "DNU". Second bit can come later but you're already touching most of the affected `.shade` files.

Might as well make everything uppercase since case isn't relevant on Windows and uppercase seems more common in KoreBuild.